lcls-twincat-motion/Library/DUTs/DUT_MotionStage.TcDUT

```diff
@@ -9,13 +9,22 @@ STRUCT
 	bLimBwd AT %I*: BOOL;
 	bHome AT %I*: BOOL;
 	bBrake AT %Q*: BOOL;
+	nEncCount	AT %I*	:	ULINT;
+	
+	//Psuedo-hardware
+	bLimGntFwd	:	BOOL := FALSE; //Forward virtual gantry limit switch
+	bLimGntBwd	:	BOOL := FALSE; //Reverse virtual gantry limit switch
 	
 	// Settings
 	bPowerSelf: BOOL:=TRUE;
 	nEnableMode: INT:=ENUM_StageEnableMode.STAGE_ENABLE_MOTION;
 	nBrakeMode: INT:=ENUM_StageBrakeMode.STAGE_BRAKE_DISABLED;
 	nHomingMode: INT:=ENUM_EpicsHomeCmd.EPICS_MOTOR_HOME_NONE;
 	
+	bGantryAxis	:	BOOL := FALSE; //Set true to activate gantry EPS
+	fGantryTol	:	LREAL := 0; //Set to gantry difference tolerance
+	nEncRef		:	ULINT	:=0;//Encoder count at which this axis is aligned with other axis
```

Comment:
Also the inconsistent comment spacing wrecks my OCD. Can we always do it like:

`code code code; // Text`
with a blank line before and after the `//` block?